langs/racket/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM alpine:3.20 AS builder
 
-RUN apk add --no-cache build-base curl git
+RUN apk add --no-cache build-base curl
 
 RUN curl -L https://github.com/racket/racket/archive/refs/tags/v8.14.tar.gz \
   | tar xz --strip-components 1
 
-RUN make -j`nproc` base
+RUN make bc -j`nproc` PKGS="" RACKETBC_SUFFIX=""
```

Comment:
I went for it and used `make bc` to initiate the build process.

This image has shrunk approximately three times, and we can _get rid of git, get it?_

```bash
oh@Yewzir:~/code-golf$ docker build --no-cache -t codegolf/lang-racket langs/racket
[+] Building 1183.7s (13/13) FINISHED
...
oh@Yewzir:~/code-golf$ docker run -it --rm codegolf/lang-racket
Welcome to Racket v8.14 [cs].
```
```bash
oh@Yewzir:~/code-golf$ docker build --no-cache -t codegolf/lang-racket langs/racket
[+] Building 634.7s (13/13) FINISHED
...
oh@Yewzir:~/code-golf$ docker run -it --rm codegolf/lang-racket
Welcome to Racket v8.14 [bc].
```
> By default, the executables for the Racket BC variant all have a `bc` or `BC` suffix, and they coexist with a Racket CS build by keeping compiled files in a `"bc"` subdirectory of the `"compiled"` directory. You can remove the `bc` suffix and the subdirectory in `"compiled"` by providing `RACKETBC_SUFFIX=""` to `make bc`.

So that should bring full enlightenment on what in the hell I am doing. 🤣